### ISSUE TITLE
Team Profiles vs Team Config

### ISSUE THREAD

**Issue by dkelosky** [zowe/docs-site#3235]
Most often, I hear Zowe CLI configuration is done via "team config".  However, [here](https://docs.zowe.org/v2.6.x/user-guide/cli-using-using-team-profiles) the configuration seems to be called "team profiles".  Later they're called also referred to as [team configuration](https://docs.zowe.org/v2.6.x/user-guide/cli-using-initializing-team-configuration) and also as ["team profile configuration files"](https://docs.zowe.org/v2.6.x/user-guide/cli-using-initializing-team-configuration#creating-team-profile-configuration-files).

Should the doc be updated for a consistent name on Zowe CLI and Zowe Explorer configuration files, that is "Team Config"?

**Comment by balhar-jakub** [comment]
Yes please, and we should probably also link somewhere to this article: https://medium.com/zowe/enterprise-rollout-of-zowe-cli-2b0a84357de3

**Comment by anaxceron** [comment]
@dkelosky Yes, I agree that there are multiple ways of describing the same concept when it comes to team config files. 

Idk that I'd agree that _one_ way bubbles up as the most popular b/c I think that depends on context. 

There are times when we discuss the overall concept of team configuration and then referring to this as "team config" feels just fine. But then there are instances when we want to discuss the different team profiles located within the same configuration file and then it makes more sense to start using the phrase "team profiles." The former is an idea, the latter is a specific set of code/variables/options.

Sometimes, when I document, I really want my user to understand that one thing and another can mean the same thing -- both the concept and the specific options. So I end up doing the most inelegant thing and lump all the words together: team profile configuration files. It _is_ intentional and, no, I don't love it.  But I do think this is more clear to the reader (that's still learning their way) than using "team config" for everything. 

It's really important to be specific, especially for that user's who still fumbling to grasp the difference between team config and user config,  not to mention global configs and project configs. If you think there are times when such specificity isn't required, send me some examples and I'll have a think. I do plan a reorg of this content in the future, so any feedback is welcome.

Lastly. I noticed that a couple of your links were to older v2 releases. Not sure if you're pointing out that the docs for the older releases are confusing (b/c the doc has been updated since then), or whether your point applies across all releases, including our latest.

@balhar-jakub I like the blog and I'll be adding a link to it on one of the team profile/config file pages.
